projects/ngx-skeleton-loader/src/lib/ngx-skeleton-loader.component.spec.ts

```diff
@@ -111,7 +111,7 @@ describe('NgxSkeletonLoaderComponent', () => {
     it('should console errors if `appearance` is an invalid option and is running in development mode', () => {
       expect(console.error).toHaveBeenCalledWith(
         // eslint-disable-next-line max-len
-        `\`NgxSkeletonLoaderComponent\` need to receive 'appearance' as: circle or line or empty string. Forcing default to "''".`,
+        `\`NgxSkeletonLoaderComponent\` need to receive 'appearance' as: circle or line or customContent or empty string. Forcing default to "''".`,
```

Comment:
Could you also added a new test scenario checking if the element is using `customContent` as expected, please?